core/modules/wsched/BlendScheduler.cc

```diff
@@ -141,7 +141,13 @@ void BlendScheduler::queCmd(util::Command::Ptr const& cmd) {
     // Check for scan tables
     SchedulerBase::Ptr s{nullptr};
     auto const& scanTables = task->getScanInfo().infoTables;
-    if (scanTables.size() > 0) {
+    bool interactive = task->getScanInteractive();
+    if (scanTables.size() <= 0 || interactive) {
```

Comment:
Does this give a warning about comparing signed and unsigned? I assume `size()` cannot return negative number, so this should be equivalent to `scanTables.size() == 0` or `scanTables.empty()`, I think latter is preferred and more idiomatic.